Mathlib/CategoryTheory/Monad/Adjunction.lean

```diff
@@ -361,4 +396,10 @@ instance (priority := 100) monadicOfReflective [Reflective R] :
   eqv := { full := Reflective.comparison_full _ }
 #align category_theory.monadic_of_reflective CategoryTheory.monadicOfReflective
 
+/-- Any coreflective inclusion has a comonadic left adjoint. -/
```

Comment:
Is this the same theorem in [Riehl]? In any case, can you add a reference to [Riehl] like above?

Reply:
This is just the dual statement, so I think it's appropriate to also refer to 5.3.3. I will add a comment.